features/0335-http-over-didcomm/README.md

```diff
@@ -0,0 +1,198 @@
+# 0335: HTTP Over DIDComm
+- Authors: [Filip Burlacu](filip.burlacu@securekey.com) (SecureKey)
+- Status: [PROPOSED](/README.md#proposed)
+- Since: 2019-12-03
+- Status Note: implementation is being explored by SecureKey  
+- Supersedes:
+- Start Date: 2019-11-26
+- Tags: feature, protocol
+
+## Summary
+
+Allows HTTP traffic to be routed over a DIDComm channel, so applications built to communicate over HTTP can make use of DID-based communication.
+
+## Motivation
+
+This protocol allows a client-server system that doesn't use DIDs or DIDComm to piggyback on top of a DID-based infrastructure, gaining the benefits of DIDs by using agents as HTTP proxies.
+
+Example use case: 
+Carl wants to apply for a car loan from his friendly neighborhood used car dealer. The dealer wants a proof of his financial stability from his bank, but he doesn't want to expose the identity of his bank, and his bank doesn't want to develop a custom in-house system (using DIDs) for anonymity. HTTP over DIDComm allows Carl to introduce his car dealer to his bank, using Aries agents and protocols, while all they need to do is install a standard agent to carry arbitrary HTTP messages.
+
```

Comment:
TODO: Further use cases, and further explanation of the use cases, were requested in the 18/12/19 morning call.